src/controllers/fs.js

```diff
@@ -195,11 +195,19 @@ class FileSystem {
     }
 
     delete(path, next) {
-        // Safety - prevent deleting the main folder.
-        if (Path.resolve(this.server.path(path)) === this.server.path()) {
-            return next(new Error('You cannot delete your home folder.'));
+        var res = path.split(",")
+        var i;
```

Comment:
`var` is not used in this project, please use `let` here, and `const` where needed elsewhere.